helm-charts/FATE-Exchange/values.yaml

```diff
@@ -31,14 +34,38 @@ modules:
     # - partyId: 10000
       # partyIp: 192.168.10.1
       # partyPort: 30101
+  nginx:
+    include: false
+    type: NodePort
+    httpNodePort:  30003
+    grpcNodePort:  30008
+    route_table:
+      # 10000: 
```

Comment:
It is better to explain these comments.
like "the fateflow service of party 10000"